.github/workflows/ci_coverage.yml

```diff
@@ -0,0 +1,139 @@
+name: SeqAn3 CI on Linux
+
+on:
+  push:
+    branches:
+      # Push events to branches matching refs/heads/master
+      - 'master'
+      # Push events to branches matching refs/heads/release*
+      - 'release*'
+  # Trigger after PR was unlabeled
+  pull_request:
+    types:
+      - unlabeled
+  # Enables a manual trigger, may run on any branch
+  workflow_dispatch:
+
+# Do not cancel on push events
+concurrency:
+  group: coverage-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: false
+
+env:
+  CMAKE_VERSION: 3.10.3
+  SEQAN3_NO_VERSION_CHECK: 1
+  TZ: Europe/Berlin
+
+defaults:
+  run:
+    shell: bash -ex {0}
+
+jobs:
+  build:
+    name: ${{ matrix.name }}
+    runs-on: ubuntu-22.04
+    timeout-minutes: 120
+    if: github.repository_owner == 'seqan' || github.event_name == 'workflow_dispatch' || github.event.label.name == 'clang-format'
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - name: "Coverage gcc12"
+            cxx: "g++-12"
+            cc: "gcc-12"
+            build: coverage
+            build_type: Debug
+
+    steps:
+      # How many commits do we need to fetch to also fetch the branch point?
+      - name: Get fetch depth
+        id: fetch_depth
+        run: echo "::set-output name=depth::$(( ${{ github.event.pull_request.commits }} + 1 ))"
+
+      - name: Checkout SeqAn3
+        uses: actions/checkout@v2
+        with:
+          path: seqan3
+          fetch-depth: ${{ steps.fetch_depth.outputs.depth }}
+          submodules: true
+
+      - name: Checkout SeqAn2
```

Comment:
why do we need seqan2? I thought it's only needed in the benchmarks

Reply:
Might be included in some seqan3/test headers I think